code_compare_template.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Is this file actually being used? Cannot find a reference

Reply:
Used here:https://github.com/SalesforceCommerceCloud/sfcc-ci/pull/174/files#diff-3e98ae93aa4f2aad8953d2195129cc5a65502df35f4ede3f254a429a3729cbebR75. I used it as a template for the code compare results. This is just a basic HTML file where I print results